### PR TITLE
IGNITE-17569: Add option to display plain tables

### PR DESCRIPTION
**Pull request by Flaugh24**
https://issues.apache.org/jira/browse/IGNITE-17569

### CODE REVIEW

File: modules/cli/src/test/java/org/apache/ignite/internal/cli/util/PlainTableRendererTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    @Test
+    void testRender() {
+        String[] header = new String[]{"id", "name", "address"};
+        Object[] row1 = new Object[]{1, "John", null};
+        Object[] row2 = new Object[]{2, "Jessica", "any address"};
+        Object[][] content = new Object[][]{row1, row2};
+        String render = PlainTableRenderer.render(header, content);
+        String[] renderedRows = render.split(System.lineSeparator());
+        Assertions.assertEquals(3, renderedRows.length);
```
Comment (PakhomovAlexander) [inline modules/cli/src/test/java/org/apache/ignite/internal/cli/util/PlainTableRendererTest.java:33]: We use static import for assertions

Comment (PakhomovAlexander) [reply modules/cli/src/test/java/org/apache/ignite/internal/cli/util/PlainTableRendererTest.java:33]: And assertAll is a good option to assert all statements.

File: modules/cli/src/test/java/org/apache/ignite/internal/cli/util/PlainTableRendererTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        for (String renderedRow : renderedRows) {
```
Comment (PakhomovAlexander) [inline modules/cli/src/test/java/org/apache/ignite/internal/cli/util/PlainTableRendererTest.java:34]: Loop-in tests are not the best choice. The test should tend to be as strict as possible.

File: modules/cli/src/main/java/org/apache/ignite/internal/cli/util/PlainTableRenderer.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+ */
+
+package org.apache.ignite.internal.cli.util;
+
+import java.util.StringJoiner;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Utility class for the plain rendering of table structures.
```
Comment (PakhomovAlexander) [inline modules/cli/src/main/java/org/apache/ignite/internal/cli/util/PlainTableRenderer.java:25]: Utility class always smells. I would recommend using Object with the constructor.

**Review by PakhomovAlexander — APPROVED** [review]

**Comment by PakhomovAlexander** [comment]
Could you please put the link to the issue and provide a short description of what this PR does.

**Comment by Flaugh24** [comment]
Added '--plain' option to 'sql' and 'topology' commands.
https://issues.apache.org/jira/browse/IGNITE-17569

**Comment by Flaugh24** [comment]
Added `--plain` option to `sql` and `topology` commands. 
https://issues.apache.org/jira/browse/IGNITE-17569

File: modules/cli/src/main/java/org/apache/ignite/internal/cli/decorators/TopologyDecorator.java [hunk 4]
```diff
@@ -35,16 +38,18 @@ public class TopologyDecorator implements Decorator<List<ClusterNode>, TerminalO
...
-                        node.getName(),
-                        node.getAddress().getHost(),
-                        String.valueOf(node.getAddress().getPort()),
-                        node.getAddress().getConsistentId() == null ? "(empty)" : node.getAddress().getConsistentId(),
-                        node.getId()
-                }
+        return () -> FlipTable.of(HEADERS, topologyToContent(topology));
+    }
+
+    protected String[][]  topologyToContent(List<ClusterNode> topology) {
```
Comment (SammyVimes) [inline modules/cli/src/main/java/org/apache/ignite/internal/cli/decorators/TopologyDecorator.java:44]:
```suggestion
    protected String[][] topologyToContent(List<ClusterNode> topology) {
```

**Review by SammyVimes — CHANGES_REQUESTED** [review]

**Review by SammyVimes — APPROVED** [review]
LGTM

**Pull request merged**